code/modules/organs/internal/heart.dm

```diff
@@ -139,7 +139,7 @@
 		var/blood_max = 0
 		var/open_wound
 		var/list/do_spray = list()
-		for(var/obj/item/organ/external/temp in owner.organs)
+		for(var/obj/item/organ/external/temp in owner.bad_external_organs)
```

Comment:
i think this reduces overall checks human process has to do, since only bad external organs will have these values. someone double check

Reply:
You'd have to make sure that this list is updated properly.

Reply:
yeah it's updated properly i believe, otherwise everything else would break